lib/pos_blockchain.go

```diff
@@ -76,17 +76,33 @@ func (bc *Blockchain) processBlockPoS(block *MsgDeSoBlock, currentView uint64, v
 		return false, false, nil, errors.Wrap(err, "processBlockPoS: Problem applying new tip: ")
 	}
 
-	// If the incoming block is not applied as the new tip, we return early.
-	if !appliedNewTip {
-		return false, false, nil, nil
+	// 5. Commit grandparent if possible. Only need to do this if we applied a new tip.
+	if appliedNewTip {
+		if err = bc.runCommitRuleOnBestChain(); err != nil {
+			return false, false, nil, errors.Wrap(err, "processBlockPoS: error committing grandparents: ")
```

Comment:
nit: technically this can commit any number of blocks that are ancestors of the grandparent block
```suggestion
			return false, false, nil, errors.Wrap(err, "processBlockPoS: error running commit rule: ")
```